processes/burntmapping_chunks.json

```
@@ -0,0 +1,196 @@
+{
+  "namespace": "user",
+  "description": "Classical Normalized Burnt Ratio(NBR) difference is performed using VITO backend on a chunk polygon.",
+  "id": "NBR_chunk_polygons",
+  "summary": "Burnt area mapping using chunk_polygon on UDF",
```

```suggestion
  "summary": "Burnt area mapping",
```

Technical details better go into the description.